TemplePlus/gamesystems/gamelib_save.cpp

```diff
@@ -211,6 +211,14 @@ bool GameSystems::SaveGame(const string& filename, const string& displayName) {
 
 	tio_fclose(file);
 
+	{
+		// user savehook
+		auto saveHookArgs = Py_BuildValue("(s)", filename.c_str());
+		pythonObjIntegration.ExecuteScript("templeplus.savehook", "save", saveHookArgs);
+		Py_DECREF(saveHookArgs);
+	}
+
+
```

Comment:
What do you think about moving this down to directly before `// Create savegame archive`?

Reply:
Also: I'd probably create an absolute path for `"Save\\Current"` (which is passed to SaveGameArchive), and pass that to the hook so it can add its own files to the archive more easily.